squareup/src/api/catalog_api.rs

```diff
@@ -157,7 +157,7 @@ impl CatalogApi {
         &self,
         params: &ListCatalogParameters,
     ) -> Result<ListCatalogResponse, SquareApiError> {
-        let url = format!("{}/list{}", &self.url(), params.to_query_string()).replace("\"", "");
+        let url = format!("{}/list{}", &self.url(), params.to_query_string());
```

Comment:
Without this replace you the api call fails because it buts an extra set of quotes around the prams.

Reply:
Yeah I'm putting it in the to_query_string() function to make the api function a little cleaner. I was writing a custom deserializer but ehhh, I don't want to maintain that

Reply:
I'll be pretty busy this weekend but if you want me to keep working on the checkout api, I will get back to it next weekend. Idk what your plans are but good work!

Reply:
Ive just been working with my local branch I will probably get to implementing the checkout api on my website this weekend or sometime next week. If I run into any issues Ill just report them here as my fork has diverged.